server/src/test/java/org/cloudfoundry/identity/uaa/login/HomeControllerViewTests.java

```diff
@@ -83,6 +84,7 @@ void tearDown() {
     }
 
     @Test
+    @Disabled
```

Comment:
Please revert the disabled test changes - they are from the "branding" branch.

Reply:
fixed.